Geometry/TrackerGeometryBuilder/src/TrackerGeomBuilderFromGeometricDet.cc

```diff
@@ -256,10 +256,10 @@ void TrackerGeomBuilderFromGeometricDet::buildGeomDet(TrackerGeometry* tracker)
         tracker->addDetId(composedDetId);
 
       } else if (gduTypeName.find("Lower") != std::string::npos) {
-        //FIXME::ERICA: the plane builder is built in the middle...
-        PlaneBuilderForGluedDet::ResultType plane = gluedplaneBuilder.plane(composed);
+        //The plane is *not* built in the middle, but on the Lower surface
+        Plane* plane = new Plane(dus->surface());
```

Comment:
Started to look into this. When I try to do something like this 
`Plane* plane = std::make_unique<Plane(dus->surface())>;`

I end up with these errors:

` error: temporary of non-literal type 'Plane' in a constant expression`

Reply:
do not understand neither Ianna comments not all the changes proposed.
Just stick to the code above (for stereo). At the limit try to understand the logic of the code above (that is there since orca times I suppose).